examples/bits.rs

```diff
@@ -11,14 +11,32 @@
 
 // The `bitvec` crate provides a lot of types; we only need a few:
 use bitvec::prelude::*;
-    // This trait provides the `.bits::<_>()` method which we use on Rust types
-    // to make `bitvec` views,
 use turtle::Turtle;
 
-/// Set the line length for each displayed bit
+// Modify these constants to change the behavior of the example.
+
+/// This controls the width of the drawn line for each bit.
 const BIT_WIDTH: f64 = 20.0;
-/// Set the vertical spacing between successive rows of bits
+
+/// This controls the vertical spacing between rows of bit lines.
 const BIT_HEIGHT: f64 = 10.0;
+
+/// This text will be inspected as individual bytes, and drawn on the screen.
+/// You can change it to see what different text looks like when viewed as bits.
+///
+/// The example program will print more information about the parts of the text
+/// to the console while the turtle draws, so that you can see how each glyph
+/// corresponds to parts of the rendered memory.
+static TEXT: &str = "¡Hola, mundo! 🌍🌏🌎";
+
+/// This number will have its bit pattern printed on screen. Rust provides some
+/// interesting numbers in its standard library; you can replace this with other
+/// numbers to see what they look like. Pi is provided as the default solely
+/// because it is well-known, and has an interesting pattern.
+const NUMBER: f64 = std::f64::consts::PI;
```

Comment:
This example focuses on viewing the bits of data, so I thought it would be a good idea to make the actual data being shown more prominently displayed in file, rather than being bound in the middle of `main` surrounded by the rest of the operational logic. Placing the data and main formatting directives together also centralizes the parts of the program that can be easily changed with immediate effect, so that users can have a fast modify/run cycle without searching for the individual declaration sites.

Reply:
Sounds great! Thanks for putting in the thought! :smile: